pandas/core/indexes/range.py

```diff
@@ -169,9 +170,16 @@ def _data(self):
         return np.arange(self.start, self.stop, self.step, dtype=np.int64)
 
     @cache_readonly
-    def _int64index(self) -> Int64Index:
+    def __int64index(self) -> Int64Index:
```

Comment:
can you rename these to something else, maybe

__int64index -> _create_int64index
_int64index -> _wrap_int64index

maybe not much better, but

Reply:
good call